src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.big-search-bar {
+  margin: 20px 0;
+}
+
+.img {
+  position: absolute;
```

Comment:
position is a property, which is not recommended to move separately since when you want to recheck your code and you try to understand why your elements behave like so, it will be hard to find out where this property is written

just leave it on those elements you need to position absolutely